spec/acceptance/self_service_spec.rb

```diff
@@ -112,6 +112,19 @@ class {'self_service':
         expect(result.stdout).to match(%r{false})
         run_shell('puppet resource service pe-postgresql ensure=running')
       end
+      it 'if S0012 conditions for false are met' do
+        run_shell('service puppet stop; puppet config set runinterval 20')
+        result = run_shell('facter -p self_service.S0012')
+        expect(result.stdout).to match(%r{false})
+        run_shell(' puppet config set runinterval 1800;service puppet start')
+      end
+      it 'if S0013 conditions for false are met' do
+        run_shell('cp $(puppet config print lastrunfile) $(puppet config print lastrunfile).bk;
```

Comment:
Small performance improvement since `puppet config print lastrunfile` takes a few seconds and we are running it multiple times.
`export lastrunfile=$(puppet config print lastrunfile) && cp ${lastrunfile}{,.bk} && sed -i \'/catalog_application/d\' $lastrunfile`